### ISSUE TITLE
CLI flags for setup command

### ISSUE THREAD

**Issue by M0NsTeRRR** [checktheroads/hyperglass#98]
# Feature Description
Add flags to setup command to run in automated mode

### Is your feature request related to a problem? Please describe.
I'm trying to automate hyperglass deployement with an ansible role unfortunately it's dirty.
`hyperglass setup -d` is the only automated mode but it install hyperglass in /home/<user> and install systemd service and I want the opposite

Best regards,

**Comment by checktheroads** [comment]
Hey @M0NsTeRRR , sorry for the delay. This is fixed, ish, in the latest version of 1.0.0-beta.72, which will be published to PyPI shortly.

In short, I removed the systemd file generation altogether for reasons noted in the [changelog](https://github.com/checktheroads/hyperglass/blob/v1.0.0/CHANGELOG.md), among others.

New docs have been added alongside the reverse proxy docs to instruct users on how to add a systemd service. I think this will work better in the long run, especially for automated installs such as yours, where systemd is not really in scope of what the hyperglass truly needs to run.

Let me know if you have any further issues.

Thanks,
Matt
